bluemira/utilities/plot_tools.py

```diff
@@ -163,13 +163,13 @@ def coordinates_to_path(x, z):
     return Path(vertices, codes)
 
 
-def set_component_plane(comp: Component, plane: Union[str, BluemiraPlane]):
-    if plane not in ["xy", "xz", "zy"] and not isinstance(plane, BluemiraPlane):
+def set_component_plane(comp: Component, plane: Union[str, BluemiraPlacement]):
```

Comment:
Just to check that we do mean that we're setting the plane of the component? It looks like the plot options have been changed to have a placement attribute but we're setting a plane (which is actually a placement) here. Should this be renamed or at least have the `plane` property changed to `placement`?

Reply:
@Dan, at present this PR is simply a "find and replace" :) For what concerns the plotting option, a `plane` should be used (so, `set_component_plane` would be fine), not a placement. I know, that created a little bit of confusion... sorry. In this PR I cannot use a `plane` for the plotting options since it is still not implemented. It is my intention to review the plotting part related to the `plane` option into the PR in which the "new" `BluemiraPlane` is implemented (to close issue #742). If you think it is better, I can create a new issue on that point (I just mentioned it in the `Additional Context`of #742 but, maybe, it was not sufficient).

Reply:
Thanks, just wanted to check. If it will come in as the solution to #742 then that's fine.